voltixApp/VoltixApp/Views/Settings/SettingsCurrencySelectionView.swift

```diff
@@ -41,7 +41,7 @@ struct SettingsCurrencySelectionView: View {
                 } label: {
                     SettingSelectionCell(
                         title: currency.rawValue,
-                        isSelected: currency==settingsViewModel.selectedCurrency
+                        isSelected: currency.description() == UserDefaults.standard.string(forKey: "currency")
```

Comment:
Lets compare SettingsLanguage objects, not strings

Reply:
btw getting UserDefaults.standard.string(forKey: "currency") inside view not the best way imo

Reply:
https://github.com/voltix-vault/voltix/issues/226